io.github.dagargo.Elektroid.yaml

```diff
@@ -0,0 +1,42 @@
+id: io.github.dagargo.Elektroid
+runtime: org.gnome.Platform
+runtime-version: '43'
```

Comment:
44 is out.

Reply:
also do you really need the GNOME runtime?

Reply:
org.gnome.Platform runtime replaced with org.freedesktop.Platform in 1be85385caad2334cc588a20fb43a5f9a4dee0d0.